src/py/flwr/cli/build.py

```diff
@@ -14,26 +14,50 @@
 # ==============================================================================
 """Flower command line interface `build` command."""
 
+import hashlib
 import os
+import shutil
+import tempfile
 import zipfile
 from pathlib import Path
-from typing import Annotated, Optional
+from typing import Annotated, Any, Optional, Union
 
 import pathspec
 import tomli_w
 import typer
 
+from flwr.common.constant import FAB_ALLOWED_EXTENSIONS, FAB_DATE, FAB_HASH_TRUNCATION
+
 from .config_utils import load_and_validate
 from .utils import get_sha256_hash, is_valid_project_name
 
 
-# pylint: disable=too-many-locals
+def write_to_zip(
+    zipfile_obj: zipfile.ZipFile, filename: str, contents: Union[bytes, str]
+) -> zipfile.ZipFile:
+    """Set a fixed date and write contents to a zip file."""
+    zip_info = zipfile.ZipInfo(filename)
+    zip_info.date_time = FAB_DATE
```

Comment:
@chongshenng Could you explain what this fixed date is for?

Reply:
Oh I found the answer (it is just a temporary placeholder)

Reply:
Yes! We're hardcoding it for now, until we decide whether the date should be set in `flwr new` or updated by the user.